composer.json

```diff
@@ -26,7 +26,8 @@
         "pimple/pimple": "1.*",
         "symfony/dependency-injection": "2.*",
         "zendframework/zend-di": "2.*",
-        "zendframework/zend-servicemanager": "2.*"
+        "zendframework/zend-servicemanager": "2.*",
+        "league/container": "1.*"
```

Comment:
@inverse 

If you want to maintain alphabetical order here, which I think we have already, you should use

```
$ composer require --dev --sort-packages league/container:1.*
```